draft-ietf-quic-transport.md

```diff
@@ -3515,6 +3515,10 @@ A server MAY send a CONNECTION_CLOSE frame with error code PROTOCOL_VIOLATION in
 response to an Initial packet it receives from a client if the UDP datagram is
 smaller than 1200 bytes. It MUST NOT send any other frame type in response, or
 otherwise behave as if any part of the offending packet was processed as valid.
+Note that a middlebox might modify datagram boundaries when multiple QUIC
+packets have been coalesced into a single UDP datagram, causing the handshake to
+fail. This is considered a failure of the path to support datagrams of the size
```

Comment:
I'm not sure this last sentence is entirely true, or necessary.  It looks to the client like its packets didn't get through, and there are many reasons that might happen (including path MTU); it looks to the server like the client is violating the protocol by sending undersized Initials.

Reply:
I tend to agree with both the sentence, as well as your statements. :) 
Really what I'm looking for is "this is a failure on the part of the network and something we've decided to live with and if someone on the path does this, it's their fault" -- better wording welcome!

Reply:
Removed this in the proposed text to go into 2925.